native.js

```diff
@@ -1034,3 +1034,22 @@ Native["com/nokia/mid/impl/jms/core/Launcher.handleContent.(Ljava/lang/String;)V
     }));
 };
 
+function UnimplementedNative(signature, returnValue) {
+    var doNotWarn;
+
+    if (typeof returnValue === "function") {
+      doNotWarn = function() { return returnValue() };
+    } else if (typeof returnValue !== "undefined") {
+      doNotWarn = function() { return returnValue };
+    } else {
+      doNotWarn = function() { };
+    }
```

Comment:
nit: Isn't this anonymous function going to return `undefined` anyway? Maybe we could collapse this into the previous `else if` clause as just an `else`